app/extensions/ember-table/template.hbs

```diff
@@ -1,5 +1,5 @@
 
-<table class="ui unstackable table {{or @tableColor 'blue'}}">
+<table class="ui unstackable table {{@tableColor}}">
```

Comment:
Won't this apply blue to all tables?

Reply:
No

Reply:
bcz we have to color in Default for getting color on top of table and that is past only in case of videoroom/list

Reply:
Sorry, I was seeing before part of the diff